bigquery/system-test/tables.test.js

```diff
@@ -191,6 +188,20 @@ describe('bigquery:tables', function () {
     });
   });
 
+  describe('browseRows', function () {
+    it('should display rows in a table', function (done) {
+      program.browseRows(options.dataset, options.table, function (err, rows) {
+        assert.equal(err, null);
+        assert.notEqual(rows, null);
```

Comment:
If we're asserting that `rows` is an array (asserting a type), then I think it's redundant to assert that `rows` is not `null` (asserting a type).